### ISSUE TITLE
Currently, does this branch support quic?

### ISSUE THREAD

**Issue by foxriver1025** [intel/asynch_mode_nginx#75]
Currently, does this branch support quic?

**Comment by Yogaraj-Alamenda** [comment]
No.. We have a separate [Experimental QUIC Support for NGINX](https://cdrdv2.intel.com/v1/dl/getContent/737522)

**Comment by foxriver1025** [comment]
@Yogaraj-Alamenda Thanks.Which client tool can test quic's handshake performance?
